$MirOS: ports/multimedia/livestream/patches/patch-liveMedia_ServerMediaSession_cpp,v 1.1.7.1 2005/03/18 15:50:47 tg Exp $

XXX can this change the protocol?

--- liveMedia/ServerMediaSession.cpp.orig	Thu May 12 01:40:47 2005
+++ liveMedia/ServerMediaSession.cpp	Thu May 19 19:59:51 2005
@@ -236,7 +236,7 @@ char* ServerMediaSession::generateSDPDes
     
     char const* const sdpPrefixFmt =
       "v=0\r\n"
-      "o=- %ld%06ld %d IN IP4 %s\r\n"
+      "o=- %lld%06ld %d IN IP4 %s\r\n"
       "s=%s\r\n"
       "i=%s\r\n"
       "t=0 0\r\n"
@@ -263,7 +263,7 @@ char* ServerMediaSession::generateSDPDes
     
     // Generate the SDP prefix (session-level lines):
     sprintf(sdp, sdpPrefixFmt,
-	    fCreationTime.tv_sec, fCreationTime.tv_usec, // o= <session id>
+	    (long long)fCreationTime.tv_sec, fCreationTime.tv_usec, // o= <session id>
 	    1, // o= <version> // (needs to change if params are modified)
 	    ourIPAddressStr, // o= <address>
 	    fDescriptionSDPString, // s= <description>
